src/create_disk.sh

```diff
@@ -561,6 +561,7 @@ fstrim -a -v
 for fs in $rootfs/boot $rootfs; do
     mount -o remount,ro $fs
     xfs_freeze -f $fs
+    xfs_freeze -u $fs
```

Comment:
OOOOH.  This could be the cause of other bugs we've seen.  This might be a candidate for backporting to older `rhcos` branches.